CRFModuleValidation/CRFMotionRecorder.swift

```diff
@@ -161,14 +165,14 @@ public class CRFMotionRecorder : RSDSampleRecorder {
     }
     
     func recordAccelerometerSample(_ data: CMAccelerometerData) {
-        let sample = CRFAccelerometerRecord(startUptime: startUptime, stepPath: currentStepPath, data: data)
+        let sample = CRFMotionRecord(startUptime: startUptime, stepPath: currentStepPath, data: data)
         self.writeSample(sample)
     }
     
     func startDeviceMotion(with motionManager: CMMotionManager, updateInterval: TimeInterval, completion: ((Error?) -> Void)?) {
         motionManager.stopDeviceMotionUpdates()
         motionManager.deviceMotionUpdateInterval = updateInterval
-        motionManager.startDeviceMotionUpdates(to: OperationQueue()) { [weak self] (data, error) in
+        motionManager.startDeviceMotionUpdates(using: .xMagneticNorthZVertical, to: OperationQueue()) { [weak self] (data, error) in
```

Comment:
Using xMagneticNorthZVertical may require manual calibration beforehand, so historically we've avoided it and instead used the xArbitraryCorrectedZVertical or just xArbitraryZVertical frame of reference (the "corrected" one uses the magnetometer, if present and calibrated, as a check to keep the arbitrary x direction from drifting over time).

Reply:
So... there are a number of settings that we collect that aren't valid *unless* this is set to magnetic north (heading, magnetic field). If we don't care about those, no worries, but we shouldn't be trying to record them.

We went with magnetic north b/c that allows android and iOS to be using similar frames. For the purposes of CRF, I am not sure why we care which direction is North (I already turned off all motion sensors for the 12 minute run b/c they are hard on the battery.

If Android can get raw gyroscope, we might want that instead (only record gravity and user acceleration from device motion - those are still interesting and do not require turning on the magnetometer).